online_carolapp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+
+FROM totvslabs/pycarol:2.30.0
+
+env SRCFOLDER=online_carolapp
+
+WORKDIR /app
+ADD requirements.txt /app/
+ADD online_carolapp/ /app/online_carolapp/
+
```

Comment:
install before adding the files to use cache when rebuilding.

Reply:
Ok, I already replicated this change to batch and online. Here and in the cookie-cutter project.